homework-submission/react-hyf/react-hyf-client/src/React_HW1/package1.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
I see no need for this package file, And it's name isn't right to work the correct way as npm expect.

Reply:
I just copy paste all the files from the previous homework. That is the main reason for that i will remove all the unnecessary files from there.